Filter/ClassFilter.php

```diff
@@ -13,10 +13,11 @@
 
 use Sonata\AdminBundle\Datagrid\ProxyQueryInterface;
 use Sonata\CoreBundle\Form\Type\EqualType;
-use Symfony\Component\Form\Extension\Core\ChoiceList\ChoiceList;
 
 class ClassFilter extends Filter
 {
+    const OPTION_SUB_CLASSES = 'sub_classes';
```

Comment:
No `const` in any other `Filter` class. It could be better to use an `OptionResolver` for options. Anyway, this does not belong in this PR.